core/variant_op.cpp

```diff
@@ -477,7 +477,13 @@ void Variant::evaluate(const Operator& p_op, const Variant& p_a, const Variant&
 					DEFAULT_OP_FAIL(MATRIX32);
 					DEFAULT_OP_LOCALMEM(+,VECTOR3,Vector3);
 					DEFAULT_OP_FAIL(PLANE);
-					DEFAULT_OP_FAIL(QUAT);
+					case QUAT: {
```

Comment:
Can't it be declared the same way as for `Vector3`, using `DEFAULT_OP_LOCALMEM(+,QUAD,Quad);`?